### PR TITLE
Articles

### PR DESCRIPTION
**Pull request by dfischer4623**
Integration of API for articles and Index.html is completed in branch: articles.

### CODE REVIEW

**Comment by dfischer4623** [comment]
Please review.  I will merge or you can merge after your review.

File: index.html
Comment (BohdiCave) [inline index.html]: I actually already included this link to jQuery on line 142/147 below, so you can just remove this line here.

File: index.html
Comment (BohdiCave) [inline index.html]: Generally, as I understand, the suggestion is to call the scripts after all the HTML code - the last thing to do before </body> tag. If you move this link below, just below the jQuery link but before my own code I've included, I think that should work best. Eventually, I'll move my code to the .js file as well - once we figure out the full sequence of calls and functions.

File: index.html
Comment (BohdiCave) [inline index.html]: Maybe also remove the 4 <br> between the <article> tags. That was just a placeholder for the mock-up. Let's see how it loads and then make the styling changes necessary.

**Review by BohdiCave — CHANGES_REQUESTED** [review]
I just three brief comments, quick changes before merging. Mostly copy-paste stuff. I'll approve once you make them and we'll merge it in. Thanks!

**Review by BohdiCave — APPROVED** [review]

**Pull request merged**
